website/source/docs/agent/options.html.md

```diff
@@ -1110,6 +1110,11 @@ default will automatically work with some tooling.
   `server_name`) to set up the client for HTTP or gRPC health checks. This allows services requiring 2-way TLS to
   be checked using the agent's credentials. This was added in Consul 1.0.1 and defaults to false.
 
+* <a name="enable_central_service_config"></a><a href="#enable_central_service_config">`enable_central_service_config`</a>
+  When set, the Consul agent will merge any service registrations with centrally configured defaults, if available.
```

Comment:
What does "if available" mean? Are we referring to a Consul version or something else?

Reply:
Before centralized config you could register a service with an agent and the service definition you provided would be used without modification (more or less).

With centralized config (and when this setting is enabled) when you register a service the agent will ask the servers for any configuration for the service. If that central config exists, it will be merged with the main service definition. If it doesn't exist (its not available) the service definition will be used as is.

Maybe this would be clearer if it said:

> the Consul agent will merge any service registrations with centrally configured defaults for the service if they exist.

🤷‍♂